From 9ffaadf806c7a061f3494ebdcaec9d43da24b2bf Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@apache.org>
Date: Tue, 27 Jun 2017 23:48:16 -0700
Subject: [PATCH 2529/2848] HADOOP-14515. Addendum. Specifically configure
 zookeeper-related log levels in KMS log4j.

(cherry picked from commit aff7a3f8eddd55501366a43d2f44f531c6e8ef48)
(cherry picked from commit ddd3e7a4db392b13aae49469d167abe83945031e)

Change-Id: I654671289d878aa5484a3efd7cfbb784ae399e49
---
 .../hadoop-kms/src/main/conf/kms-log4j.properties  |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-kms/src/main/conf/kms-log4j.properties b/hadoop-common-project/hadoop-kms/src/main/conf/kms-log4j.properties
index 8e6d909..408317b 100644
--- a/hadoop-common-project/hadoop-kms/src/main/conf/kms-log4j.properties
+++ b/hadoop-common-project/hadoop-kms/src/main/conf/kms-log4j.properties
@@ -35,4 +35,7 @@ log4j.additivity.kms-audit=false
 log4j.rootLogger=ALL, kms
 log4j.logger.org.apache.hadoop.conf=ERROR
 log4j.logger.org.apache.hadoop=INFO
-log4j.logger.com.sun.jersey.server.wadl.generators.WadlGeneratorJAXBGrammarGenerator=OFF
\ No newline at end of file
+log4j.logger.com.sun.jersey.server.wadl.generators.WadlGeneratorJAXBGrammarGenerator=OFF
+# make zookeeper log level an explicit config, and not changing with rootLogger.
+log4j.logger.org.apache.zookeeper=INFO
+log4j.logger.org.apache.curator=INFO
\ No newline at end of file
-- 
1.7.9.5

